### DISCUSSION TITLE
Is there a way to know if the MS Edge WebView2 runtime is valid?

### DISCUSSION THREAD

**Discussion by koreaflash** [airsdk/Adobe-Runtime-Support#3407] [Q&A]
When I run the app after setting to use WebView2,
- If there is a WebView2 runtime, it displays the webpage using the WebView2 runtime.
- If there is no WebView2 runtime, it seems to display it using an older version of WebKit.

Is there a way to know if the WebView2 runtime is valid?
"StageWebView.isSupported" is always true.

**Comment by ajwfrost** [comment]
Hi

The "older" version is actually the IE browser engine.. available for a few more years I think.

If you don't want to display anything using that, then you can set the `<UseWebView2>` value in your app descriptor file to `exclusive`, which means it would only check/create the Edge-based WebView2..
https://airsdk.dev/docs/building/application-descriptor-files/elements/windows#usewebview2

thanks

**Reply by koreaflash** [reply]
thanks
